src/main/java/org/mifos/ops/zeebe/camel/routes/OperationsRouteBuilder.java

```diff
@@ -23,6 +23,22 @@
     @Override
     public void configure() {
 
+        from("rest:POST:/channel/workflow/")
```

Comment:
Add request response structure in comments above.

Reply:
done